### ISSUE TITLE
break url to pdf/ps

### ISSUE THREAD

**Issue by RekGRpth** [michaelrsweet/htmldoc#409]
this change in commit https://github.com/michaelrsweet/htmldoc/commit/580a1e5f4f677ab7e285a50619ea22b3c4d121dc
```diff
 
   DEBUG_printf(("file_find: \"%s\" not in web cache of %d files...\n", s, (int)web_files));
@@ -725,7 +725,13 @@ file_find(const char *path,		/* I - Path "dir;dir;dir" */
     }
   }
 
-  return (file_find_check(s));
+  if (file_find_check(s))
+  {
+    strlcpy(filename, s, sizeof(filename));
+    return (filename);
+  }
+  else
+    return (NULL);
 }
 
 
```
break url to pdf/ps

**Comment by michaelrsweet** [comment]
@RekGRpth Can you provide an example where this breaks? This change was needed to fix a value scoping issue.

**Comment by RekGRpth** [comment]
this does not work
```sh
htmldoc --bodyfont Sans --charset utf-8 --webpage -f test.pdf --header . --footer . "https://google.com"
```

**Comment by michaelrsweet** [comment]
[master 008861d] Fix crash bug with data: URIs (Issue #410)
